crates/shared/src/price_estimation/native/coingecko.rs

```diff
@@ -71,12 +72,23 @@ impl CoinGecko {
         })
     }
 
-    pub async fn send_request_price_in_eth(&self, token: Token) -> NativePriceEstimateResult {
+    pub async fn send_bulk_request_price_in_eth(
```

Comment:
There are a few function that are public but seem to be only implementation details. Can we make them private instead?